config/config.go

```diff
@@ -139,24 +142,23 @@ func UpdateGlobal(f func(conf *Config)) func() {
 	return restore
 }
 
-const (
-	globalTxnScope = "global"
-)
-
-// GetTxnScopeFromConfig extracts @@txn_scope value from config
+// GetTxnScopeFromConfig returns whether it's global scope and extracts @@txn_scope value from config.
 func GetTxnScopeFromConfig() (bool, string) {
 	if val, err := util.EvalFailpoint("injectTxnScope"); err == nil {
 		v := val.(string)
 		if len(v) > 0 {
 			return false, v
 		}
-		return true, globalTxnScope
+		return true, oracle.GlobalTxnScope
 	}
 
 	if kvcfg := GetGlobalConfig(); kvcfg != nil && len(kvcfg.TxnScope) > 0 {
-		return false, kvcfg.TxnScope
+		if kvcfg.EnableLocalTxn {
```

Comment:
How about check `kvcfg.EnableLocalTxn && len(kvcfg.TxnScope) > 0` at the same level? That will looks more readable to me.